torch/distributions/__init__.py

```diff
@@ -34,11 +34,12 @@
 from .beta import Beta
 from .categorical import Categorical
 from .cauchy import Cauchy
+from .chi2 import Chi2
 from .dirichlet import Dirichlet
 from .distribution import Distribution
 from .exponential import Exponential
 from .gamma import Gamma
-from .chi2 import Chi2
```

Comment:
Will not adding `.chi2` before `.gamma` cause circular import as @alicanb commented on my earlier PR where I did something similar?

Reply:
No this is fine. The Gamma module we be loaded (but not imported) at the line `from .chi2 import Chi2`. The import order in this PR was automatically determined by `isort` (which saves a ton of time BTW).